### PR TITLE
[Feature] Textbooks

### PR DESCRIPTION
**Pull request by Marioneq4958**

### CODE REVIEW

File: app/core/config.py [hunk 1]
```diff
@@ -11,21 +11,12 @@ class Settings(BaseSettings):
...
-  if PYTHON_VER == "3.10.4" or PYTHON_VER == "3.10.5":
-    try:
-      x = os.environ["W_BACKEND_CORS"]
-      print(f'{fg.orange}Wykryto zmienną W_BACKEND_CORS!{fg.rs}')
-      CORS_ORIGINS: str = x
-    except:
-      print(f'{fg.lightgreen}Nie wykryto zmiennej W_BACKEND_CORS. Używanie ustawień domyślnych!{fg.rs}')
-      CORS_ORIGINS: list = [ "http://localhost:8080" ]
-  else:
-    raise Exception(f'{fg.red}Twoja wersja pythona to: {PYTHON_VER} a potrzebna jest 3.10.4 lub wyższa!!!{fg.rs}')
```
Comment (Zaptyp) [inline app/core/config.py:23]: Revert this!!!

**Review by Zaptyp — CHANGES_REQUESTED** [review]
Nice!

File: app/core/config.py (hunk 1 above)
Comment (Marioneq4958) [reply app/core/config.py:23]: Why?

Comment (Marioneq4958) [reply app/core/config.py:23]: Python version 3.10.4 or higher isn't required.

Comment (Zaptyp) [reply app/core/config.py:23]: Delete the python lock

Comment (Zaptyp) [reply app/core/config.py:23]: Change try to if and except to else

**Review by Zaptyp — APPROVED** [review]

**Pull request merged**